core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -2864,8 +2876,12 @@ wasmtime_ssp_sock_send_to(wasm_exec_env_t exec_env, struct fd_table *curfds,
     __wasi_errno_t error;
     int ret;
     bh_sockaddr_t sockaddr;
+    /* make a copy because the value pointed seem to be modifed */
```

Comment:
Same as above, seems it is because that Zephyr's os_socket_send_to implementation uses the input of dest_addr?

Reply:
same as above